src/state/contents/actions.js

```diff
@@ -158,6 +158,7 @@ export const fetchContentsWithFilters = (
   quickFilterStatusParam = '',
   quickFilterOwnerGroup,
   joinGroupsToParse,
+  mode = 'full',
```

Comment:
pls make a const variables (probably located in `state/contents/const.js`) regarding MODE_FULL and MODE_LIST. Implement them here and areas using with `mode`.